warehouse/models/mart/gtfs_quality/idx_monthly_reports_site.sql

```diff
@@ -104,7 +104,28 @@ summarize_feed_info AS (
     GROUP BY 1, 2
 ),
 
-idx_monthly_reports_site AS (
+dataset_map AS (
```

Comment:
This is mostly a dbt conventions thing but I would actually:
* Move this up towards the top of the file because it uses a `ref` 
* Because we actually already reference this table once above, I would move this to the very top and then refactor `assessed_entities` to select from this with its end of month filter applied

(So, have just one CTE that directly references `ref('int_gtfs_quality__organization_dataset_map')` and then have the second CTE filter that first CTE.)

Rationale is to make "import" refs visible at the top of the file and show clearly which elements are using the same data as opposed to referencing twice. I think sometimes there can be actual query performance impacts too but in this case this is a pretty small table so I think that's less relevant.

Reply:
makes sense, made the change